README.md

```diff
@@ -19,21 +19,18 @@ Optuna-Integration API reference is [here](https://optuna-integration.readthedoc
 
 * [AllenNLP](https://optuna-integration.readthedocs.io/en/stable/reference/index.html#allennlp)  ([example](https://github.com/optuna/optuna-examples/tree/main/allennlp))
 * [BoTorch](https://optuna-integration.readthedocs.io/en/stable/reference/index.html#botorch)
-* [Catalyst](https://optuna-integration.readthedocs.io/en/stable/reference/index.html#catalyst)  ([example](https://github.com/optuna/optuna-examples/blob/main/pytorch/catalyst_simple.py))
 * [CatBoost](https://optuna-integration.readthedocs.io/en/stable/reference/index.html#catboost)  ([example](https://github.com/optuna/optuna-examples/blob/main/catboost/catboost_pruning.py))
 * [Chainer](https://optuna-integration.readthedocs.io/en/stable/reference/index.html#chainer)  ([example](https://github.com/optuna/optuna-examples/tree/main/chainer/chainer_integration.py))
 * [ChainerMN](https://optuna-integration.readthedocs.io/en/stable/reference/index.html#chainermn)  ([example](https://github.com/optuna/optuna-examples/tree/main/chainer/chainermn_simple.py))
 * [Dask](https://optuna-integration.readthedocs.io/en/stable/reference/index.html#dask) ([example](https://github.com/optuna/optuna-examples/tree/main/dask/dask_simple.py))
 * [FastAI](https://optuna-integration.readthedocs.io/en/stable/reference/index.html#fast-ai) (
-  V1 ([example](https://github.com/optuna/optuna-examples/tree/main/fastai/fastaiv1_simple.py)),
   V2  ([example](https://github.com/optuna/optuna-examples/tree/main/fastai/fastaiv2_simple.py)))
```

Comment:
```md
* [FastAI](https://optuna-integration.readthedocs.io/en/stable/reference/index.html#fast-ai) ([example](https://github.com/optuna/optuna-examples/tree/main/fastai/fastaiv2_simple.py))
```

Reply:
Done in commit https://github.com/optuna/optuna-integration/pull/114/commits/d8c59769f9884958debd371d37cf0efc6c8579d2 .